### ISSUE TITLE
Stored Object: paper:arxiv.1511.07948

### ISSUE THREAD

**Issue by dmarx** [dmarx/papers-feed-dev#5]
{
  "primary_id": "arxiv.1511.07948",
  "source": "arxiv",
  "sourceId": "1511.07948",
  "url": "https://arxiv.org/abs/1511.07948",
  "title": "Learning Halfspaces and Neural Networks with Random Initialization",
  "authors": "Yuchen Zhang, Jason D. Lee, Martin J. Wainwright, Michael I. Jordan",
  "abstract": "We study non-convex empirical risk minimization for learning halfspaces and\nneural networks. For loss functions that are $L$-Lipschitz continuous, we\npresent algorithms to learn halfspaces and multi-layer neural networks that\nachieve arbitrarily small excess risk $\\epsilon&gt;0$. The time complexity is\npolynomial in the input dimension $d$ and the sample size $n$, but exponential\nin the quantity $(L/\\epsilon^2)\\log(L/\\epsilon)$. These algorithms run multiple\nrounds of random initialization followed by arbitrary optimization steps. We\nfurther show that if the data is separable by some neural network with constant\nmargin $\\gamma&gt;0$, then there is a polynomial-time algorithm for learning a\nneural network that separates the training data with margin $\\Omega(\\gamma)$.\nAs a consequence, the algorithm achieves arbitrary generalization error\n$\\epsilon&gt;0$ with ${\\rm poly}(d,1/\\epsilon)$ sample and time complexity. We\nestablish the same learnability result when the labels are randomly flipped\nwith probability $\\eta&lt;1/2$.",
  "timestamp": "2025-03-05T07:28:32.623Z",
  "rating": "novote",
  "arxivId": "1511.07948",
  "arxiv_tags": [
    "cs.LG"
  ],
  "published_date": "2015-11-25T04:41:20Z"
}

---
*Copied from dmarx/papers-feed#1551*

**Comment by dmarx** [comment]
{
  "_data": {
    "primary_id": "arxiv.1511.07948",
    "source": "arxiv",
    "sourceId": "1511.07948",
    "url": "https://arxiv.org/abs/1511.07948",
    "title": "Learning Halfspaces and Neural Networks with Random Initialization",
    "authors": "Yuchen Zhang, Jason D. Lee, Martin J. Wainwright, Michael I. Jordan",
    "abstract": "We study non-convex empirical risk minimization for learning halfspaces and\nneural networks. For loss functions that are $L$-Lipschitz continuous, we\npresent algorithms to learn halfspaces and multi-layer neural networks that\nachieve arbitrarily small excess risk $\\epsilon&gt;0$. The time complexity is\npolynomial in the input dimension $d$ and the sample size $n$, but exponential\nin the quantity $(L/\\epsilon^2)\\log(L/\\epsilon)$. These algorithms run multiple\nrounds of random initialization followed by arbitrary optimization steps. We\nfurther show that if the data is separable by some neural network with constant\nmargin $\\gamma&gt;0$, then there is a polynomial-time algorithm for learning a\nneural network that separates the training data with margin $\\Omega(\\gamma)$.\nAs a consequence, the algorithm achieves arbitrary generalization error\n$\\epsilon&gt;0$ with ${\\rm poly}(d,1/\\epsilon)$ sample and time complexity. We\nestablish the same learnability result when the labels are randomly flipped\nwith probability $\\eta&lt;1/2$.",
    "timestamp": "2025-03-05T07:28:32.623Z",
    "rating": "novote",
    "arxivId": "1511.07948",
    "arxiv_tags": [
      "cs.LG"
    ],
    "published_date": "2015-11-25T04:41:20Z"
  },
  "_meta": {
    "client_version": "0.3.2",
    "timestamp": "2025-03-05T07:28:33.655Z",
    "update_mode": "append"
  },
  "type": "initial_state"
}

---
*Original comment by @dmarx*
